### PR TITLE
New story feature

### PR DESCRIPTION
**Pull request by Gr3gbug**
Issue 1.6 (Write a story on the rolled dice)
- Modified some typo in Die.py
- Write the needed HTML file (new_story and write_story)
- Write the method (write_story, new_stories) to manage this issue in stories.py
- Added some required fiedl in database.py

### CODE REVIEW

File: monolith/views/stories.py [hunk 1]
```diff
@@ -50,3 +56,22 @@ def _like(authorid, storyid):
     else:
         message = 'You\'ve already liked this story!'
     return _stories(message)
+
+# TODO to complete
+@stories.route('/stories/new_story', methods=['GET'])
```
Comment (sduoase) [inline monolith/views/stories.py]: This should have a POST method which creates the unpublished story in the database and then redirect to /write_story [GET], right?

I am asking to make sure that we are on the same page.

File: monolith/views/stories.py [hunk 2]
```diff
@@ -50,3 +56,22 @@ def _like(authorid, storyid):
...
+@login_required
+def new_stories():
+    dice_set = retrieve_dice_set()
+    themes = [dice_set.theme]
+
+    return render_template("new_story.html", themes=themes)
+
+# TODO to complete
+@stories.route('/write_story', methods=['POST'])
```
Comment (sduoase) [inline monolith/views/stories.py]: Following the comment on /stories/new_story, this should have a GET method as well which retrieves an unpublished story, right?

File: monolith/templates/new_story.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<html>
+
+<body>
+  <h1>Create a new story</h1>
+  <form action="/write_story" method="POST">
```
Comment (sduoase) [inline monolith/templates/new_story.html]: Following the comment on /new_story [GET] this should send a post request to /write_story [POST] and there we create the empty unpublished story in the db.

File: monolith/classes/Die.py [hunk 4]
```diff
@@ -7,7 +7,7 @@ def __init__(self, faces):
         self.faces = faces
 
     def throw(self):
-        face = random.randint(0, len(faces)) - 1
+        face = random.randint(0, len(self.faces)) - 1
```
Comment (alessandrobelfiore) [inline monolith/classes/Die.py]: This was already fixed, the brackets are not correct, check master

File: monolith/database.py [hunk 5]
```diff
@@ -87,27 +105,36 @@ def __init__(self, user, follower, *args, **kw):
...
 def retrieve_dice_set():
-    dice = db.session.query(Dice).first()
+    # dice = db.session.query(Dice).first()
+    dice = db.session.query(Dice).all()
     if dice is None:
         return None
 
     json_dice_set = dice.serialized_dice_set
-    return _deserialize_dice_set(json_dice_set)
+    dice_set = _deserialize_dice_set(json_dice_set)
```
Comment (alessandrobelfiore) [inline monolith/database.py]: This doesn't compile, cause .all() returns a list

**Pull request closed without merging**